identityClient/identityClient.go

```diff
@@ -3,6 +3,7 @@ package identityClient
 import (
 	"context"
 	"fmt"
+	"github.com/gorilla/schema"
```

Comment:
nit: std lib packages should be grouped together, this could go done to lines 13-14

Reply:
wtf goland